unified-codes/apps/web/src/components/molecules/EntityTypeFilter.tsx

```diff
@@ -0,0 +1,30 @@
+import * as React from 'react';
+import { ToggleButtonGroup } from '@unified-codes/ui';
+
+export type EntityTypeFilter = React.FunctionComponent;
+
+export const EntityTypeFilter: EntityTypeFilter = () => {
+  const [entityStates, entityStateChange] = React.useState([
+    { name: 'Drugs', active: true },
```

Comment:
I was wondering how to make use of the filter, given that all of the items we have are `medicinal_product` and thinking that someone needs to review the current dataset and codify items, as `drug` and `other`

Reply:
Yeah, I was thinking `other_products` might be suited as another node/entity type that has all the non-drug things directly linked as children in Dgraph? That way we'd be able to pull all 'other' and children when the other tab is selected.